src/main/scala/chisel3/tester/TreadleBackend.scala

```diff
@@ -162,9 +160,8 @@ class TreadleBackend[T <: MultiIOModule](dut: T,
 }
 
 object TreadleExecutive {
-  import chisel3.internal.firrtl.Circuit
   import chisel3.experimental.BaseModule
-
+  import chisel3.internal.firrtl.Circuit
```

Comment:
wat? Where does get used?

Reply:
dangling import, removed

Reply:
This just got moved one line down, it is not new, I'll move it back but you must have put it there originally